Tests/Service/OgoneFileBuilderTest.php

```diff
@@ -135,7 +135,7 @@ public function testValidRound()
     {
         $expectedTextContent = "OHL;ETSCPC;anglet64600;;userapi64600;
 OHF;FILEorder_id25;MTR;SAS;1;
-INV;EUR;;;;transactionId;LEGAL;;payId;SAS;;;;ETSCPC;;1;aliasGSP;azerty;;;1700065264;;;;;;;;order_id25;;;4403;881;5284;
+INV;EUR;;;;transactionId;LEGAL;;payId;SAS;;;;ETSCPC;;1;aliasGSP;azerty;;;1700065264;;;;;;;;order_id25;;;4403;880.6;5283.6;
```

Comment:
this should not change, do not send floats

Reply:
please add another test case to reproduce your exact issue, but this test case should not change